<?xml version="1.0" ?>
<testsuite name="ESalesTest-20201023100705" tests="1" time="12.048" failures="0" errors="0">
	<testcase classname="ESalesTest" name="test_01_check_messages" time="12.048">
		<error type="NoSuchElementException" message="Message: no such element: Unable to locate element: {&quot;method&quot;:&quot;name&quot;,&quot;selector&quot;:&quot;txtPassword&quot;}
  (Session info: headless chrome=86.0.4240.111)
  (Driver info: chromedriver=2.39.562718 (9a2698cba08cf5a471a29d30c8b3e12becabb0e9),platform=Windows NT 10.0.14393 x86_64)
"><![CDATA[Traceback (most recent call last):
  File "transus_integration_test.py", line 21, in with_screen_shot
    return fn(self, *args, **kwargs)
  File "transus_integration_test.py", line 70, in test_01_check_messages
    password = self.driver.find_element_by_name("txtPassword")
  File "C:\Program Files (x86)\Python38-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 496, in find_element_by_name
    return self.find_element(by=By.NAME, value=name)
  File "C:\Program Files (x86)\Python38-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "C:\Program Files (x86)\Python38-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Program Files (x86)\Python38-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"name","selector":"txtPassword"}
  (Session info: headless chrome=86.0.4240.111)
  (Driver info: chromedriver=2.39.562718 (9a2698cba08cf5a471a29d30c8b3e12becabb0e9),platform=Windows NT 10.0.14393 x86_64)

]]></error>
	</testcase>
	<system-out><![CDATA[]]></system-out>
	<system-err><![CDATA[]]></system-err>
</testsuite>
